src/index.js

```diff
@@ -6,6 +6,20 @@ export { default as orientation } from './orientation/index.js';
 
 export { default as requestPoolManager } from './requestPool/requestPoolManager.js';
 
+export {
```

Comment:
Maybe we should export them as cornerstoneTools.drawing.xyz ?

Reply:
Done.